pom.xml

```diff
@@ -150,15 +150,16 @@
 					<version>3.2.0</version>
 				</plugin>
 
-				<plugin>
+					<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-checkstyle-plugin</artifactId>
-					<version>${checkstyle.plugin.version}</version>
```

Comment:
Yes, I built it clean on my third attempt. I also tried deleting the checkstyle plugin, but the error remains.